### DISCUSSION TITLE
Multiome single cell RNA and ATAC sequencing

### DISCUSSION THREAD

**Discussion by bhekmat** [stuart-lab/signac#1285] [Q&A]
Hi @timoast  

I attempted to integrate a Multiome scATACseq and scRNAseq dataset (generated using Joint RNA and ATAC analysis :10xmultime vignette, https://stuartlab.org/signac/articles/pbmc_multiomic.html) for my WT and KO cells. My question for you is what would you best recommend as a method to integrate the two cell types containing these two assays?

**Comment by timoast** [comment]
Are you asking how to integrate multiple 10x Multiome datasets?

**Reply by bhekmat** [reply]
Yes. I initially created a list of the two objects (WT,KO)>Selectintegrationfeatures>Prepsctintegration>Findintegrationanchors>IntegrateData. 
I was just wondering if this method is correct and will integrate the WT and KO cells and their two (RNA, ATAC) assays.

Thank you.

**Reply by timoast** [reply]
Yes, for multiome data integration you can basically follow the same workflow that you would use for integrating scRNA-seq data. You just need to make sure you have quantified the same peaks for the scATAC assay in each object
